solr/core/src/java/org/apache/solr/servlet/HttpSolrCall.java

```diff
@@ -493,7 +493,7 @@ Action authorize() throws IOException {
     }
     if (statusCode == AuthorizationResponse.FORBIDDEN.statusCode) {
       if (log.isDebugEnabled()) {
-        log.debug("UNAUTHORIZED auth header {} context : {}, msg: {}", req.getHeader("Authorization"), context, authResponse.getMessage()); // logOk
+        log.debug("UNAUTHORIZED auth header {} context : {}, msg: {}", req.getHeader("Authorization"), context, authResponse.getMessage()); // nowarn
```

Comment:
What does our source validation complain about here?

Many of the logok/nowarn places look fine to me at a glance but I'm no match for the logging policeman ;-)